packages/terra-framework-site/src/app/site.scss

```diff
@@ -0,0 +1,91 @@
+// Styles specific to development only
+// stylelint-disable
+:local {
+  .app {
+    height: 100%;
+    position: relative;
+    width: 100%;
+  }
+
+  // .site-nav {
```

Comment:
Oops. Looks like this got missed in clean up.

Reply:
Whoops, I'll remove this.